src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -70,6 +70,12 @@
     private final List<Content> contents = new LinkedList<>();
 
     /**
+     *  A boolean flag that determines whether logging should be enabled for
+     *  this conference.
+     */
+    private final boolean loggingEnabled;
```

Comment:
This clashes with the implementation in #284 (which implements pretty much the same). Do you prefer to use this version?